Pursuit-Core-iOS-Episodes-from-Online/Pursuit-Core-iOS-Episodes-from-Online/Controllers/ViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  ViewController.swift
+//  Pursuit-Core-iOS-Episodes-from-Online
+//
+//  Created by Benjamin Stone on 9/5/19.
+//  Copyright © 2019 Benjamin Stone. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+        
+        @IBOutlet weak var showTableVIew: UITableView!
+        @IBOutlet weak var showSearchBar: UISearchBar!
+        @IBOutlet weak var spinnerOne: UIActivityIndicatorView!
+        
+        var viewShow = [TVShowsWrapper](){
+            didSet {
+                showTableVIew.reloadData()
+            }
+        }
+        
+        override func viewDidLoad() {
+            super.viewDidLoad()
+            showTableVIew.delegate = self
+            showTableVIew.dataSource = self
+            showSearchBar.delegate = self
+            loadData(word: nil)
+        }
+        var userSearchTerm: String? {
+            didSet {
+                self.showTableVIew.reloadData()
+            }
+        }
+        var filteredShow: [TVShowsWrapper]  {
+            guard let userSearchTerm = userSearchTerm else {
+                return viewShow
+            }
+            guard userSearchTerm != "" else {
+                return viewShow
+            }
+            return viewShow
+        }
+       
+
+      
+        override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+            guard let showVc = segue.destination as? EpisodesViewController else {
+                fatalError("Unexpected segue")
+            }
+            guard let selectedIndexPath = showTableVIew.indexPathForSelectedRow
+                else { fatalError("No row selected") }
+            showVc.show = filteredShow[selectedIndexPath.row]
+        }
+       private func loadData(word: String?){
```

Comment:
this Load data func does not need a word parameter. Since you will only be using this func in this VC and your getShow func already takes in a word, just pass in the word into that func directly.